### PR TITLE
Ledbar

### PR DESCRIPTION
**Pull request by alexnagelberg**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Printer mods

#### How was it tested? 
Printed using different printers and tried part on different extrusion makes

#### Any background context you want to provide?
Wanted a simple setup for an LED bar I got off amazon that would also allow me to tuck wires and made this angle corner that should be reusable for other things as it'll attach to 2020s and let you mount anything with an m3 screw to it

#### Screenshots (if appropriate)
Should be included in the images directory

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -333,3 +333,4 @@ like so:
 | | [V0 Skirt set](./Zen3D/V0_Skirt)| SSR, PowerSwitch, Nameplate and color change printing  | :heavy_check_mark: |:x: |:x: |:x:|
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
+|Nagelberg|[90 degree LED bar](./nagelberg/led_bar_90_deg)|Simple mount for attaching LED bar and angling down 90deg|:grey_question:|:grey_question:|:heavy_check_mark:|:grey_question:|:grey_question:|
```
Comment (cbc02009) [inline printer_mods/README.md:336]: Please add this in alphabetical order

File: printer_mods/nagelberg/led_bar_90_deg/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# LED Bar BOM
+
+Mount alone:
+- 2x M3 heat inset
+- 2x M3x8 SHCS
+- 2x M3 t-nut
+
+LED Bar:
+- 2x M3x6 BHCS for LED bar (any M3 will work for mounting anything else to it)
+- 1x LED bar (you can use any bar with brackets -- this mod is really simple. the one I used: [Amazon](https://www.amazon.com/gp/product/B07RNDDYYK/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1))
```
Comment (cbc02009) [inline printer_mods/nagelberg/led_bar_90_deg/README.md:12]: change this link to https://www.amazon.com/gp/product/B07RNDDYYK/ to remove all useless metadata

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Please add this in alphabetical order

**Comment by newphreak** [comment]
@alexnagelberg can you make the requested changes? :)

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**